synapse/rest/client/pusher.py

```diff
@@ -132,6 +133,23 @@ async def on_POST(self, request: SynapseRequest) -> Tuple[int, JsonDict]:
         return 200, {}
 
 
+class LegacyPushersRemoveRestServlet(RestServlet):
+    """
+    A servlet to handle legacy "email unsubscribe" links, forwarding requests to the ``UnsubscribeResource``
+    """
+
+    PATTERNS = client_patterns("/pushers/remove$", releases=[], v1=False, unstable=True)
+
+    def __init__(self, hs: "HomeServer"):
+        super().__init__()
+        self._resource = UnsubscribeResource(hs)
```

Comment:
It probably makes more sense to have this sub-class `UnsubscribeResource` and override `PATTERNS` and leave the rest blank.